src/axom/primal/operators/detail/evaluate_integral_impl.hpp

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef PRIMAL_EVAL_INTEGRAL_IMPL_HPP_
+#define PRIMAL_EVAL_INTEGRAL_IMPL_HPP_
+
+// Axom includes
+#include "axom/config.hpp"  // for compile-time configuration options
+#include "axom/primal.hpp"
+
+// MFEM includes
+#ifdef AXOM_USE_MFEM
+  #include "mfem.hpp"
+#else
+  #error "Primal's integral evaluation functions require mfem library."
+#endif
+
+// C++ includes
+#include <cmath>
+
+namespace axom
+{
+namespace primal
+{
+namespace detail
+{
+/*!
+ * \brief Computes the tangent vector to a Bezier curve.
+ * 
+ * Uses the de Casteljau algorithm. Functionality will eventually
+ * be supplanted by Bezier curve class method.
+ * 
+ * \param [in] c the Bezier curve
+ * \param [in] t on [0, 1] the parameter value on which to evaluate the derivative
+ * \return the tangent vector
+ */
+template <typename T>
+primal::Vector<double, 2> temp_dt(const primal::BezierCurve<double, 2>& c, T t)
```

Comment:
I merged #844, so please remove this function in favor of the one in the `BezierCurve` class